src/pytorch_lightning/core/module.py

```diff
@@ -1329,7 +1329,9 @@ def configure_optimizers(self):
         Note:
             Some things to know:
 
-            - Lightning calls ``.backward()`` and ``.step()`` on each optimizer and learning rate scheduler as needed.
+            - Lightning calls ``.backward()`` and ``.step()`` on each optimizer as needed.
+            - If learning rate scheduler is specified in ``configure_optimizers()`` with parameter ``"interval"``,
+              Lightning will call scheduler's ``.step()`` method automatically.
```

Comment:
```suggestion
            - If learning rate scheduler is specified in ``configure_optimizers()`` with key ``"interval"`` (default="epoch") in the scheduler configuration,
              Lightning will call the scheduler's ``.step()`` method automatically in case of automatic optimization.
```
how about?